pkg/cluster/internal/create/actions/config/config.go

```diff
@@ -79,6 +79,7 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 			ControlPlaneEndpoint: controlPlaneEndpoint,
 			APIBindPort:          kubeadm.APIServerPort,
 			Token:                kubeadm.Token,
+			PodSubnet:            "10.244.0.0/16", // TODO: make configurable
```

Comment:
Please, take a look at this approach to see if it makes sense https://github.com/kubernetes-sigs/kind/pull/348/commits/ef8d84bb33a314426e1c2705fad935191ac4b907